config/prte_configure_options.m4

```diff
@@ -408,19 +408,19 @@ PRTE_ZLIB_CONFIG
 dnl Check for FT
 AC_MSG_CHECKING([if want fault tolerance support])
 AC_ARG_ENABLE([prte-ft],
-    [AC_HELP_STRING([--enable-prte-ft],
-        [ENable PRRTE fault tolerance support (default: disabled)])])
-if test "$enable_prte_ft" = "yes"; then
-    AC_MSG_RESULT([yes])
-    prte_enable_ft=1
-    PRTE_SUMMARY_ADD([[Options]],[[Fault tolerance]], [prte_ft], [yes])
-else
+    [AC_HELP_STRING([--disable-prte-ft],
+        [Disable PRRTE fault tolerance support (default: enabled)])])
+if test "$enable_prte_ft" = "no"; then
```

Comment:
Let's leave this `off` by default for v2.0 so groups that don't need it have one clean release they can tag. We can turn it `on` by default afterwards.

Reply:
In order to get things moving, I have postponed that change to a separate PR.